index.js

```diff
@@ -0,0 +1,33 @@
+const item = {
+    itemId: '55sdife7-w32e-5135-abs9-b5632a4ba5b5',
+    name: 'Sweatpants',
+    description: 'Drake Bulldogs 2020 Sweatpants',
+    price: 24.99,
+    size: 'XS, S, M, L, XL, XXL'
+};
+
+const firstName = 'Esteban';
+const lastName = 'Julio Ricardo Montoya de la Rosa Ramirez';
+
+const customer = {
+    firstName,
+    lastName,
+    email: `${firstName.toLowerCase()}.${lastName.toLowerCase()}@drake.edu`,
+    phoneNumber: '+15155555555'
+};
+
+const cart = {
+    cartid: '68sdife7-e56d-5135-abs9-b5632a5sdf5b6',
+    userid: '68sdife7-s59r-5135-abq5-b5681d5sdf5b6'
+};
+
+const cartitems = {
```

Comment:
```js
const cartItems = [
{
cartid: '68sdife7-e56d-5135-abs9-b5632a5sdf5b6',
cartItemId: '4b6c8cfe-512a-11ea-8d77-2e728ce88125',
quantity: 3
},
{
cartid: '68sdife7-e56d-5135-abs9-b5632a5sdf5b6',
cartItemId: '5b6c8cfe-512a-11ea-9d66-2e728ce88135',
quantity: 2
}
]
```

cart items will be a list and each cart item will be part of a single cart so they will share the same cartId